### DISCUSSION TITLE
Form minimize/maximize/close buttons appear disabled

### DISCUSSION THREAD

**Discussion by washburnit** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2927] [Q&A]
Hi,
I'm facing a strange issue (probably I'm doing something wrong): in the MainWindow the three form buttons in the subject appear enabled and clickable, if I open a secondary form the same buttons are visible but "disabled", I mean if I click on one of them (close button for example) nothing happens and it appears as "grayed".
What should I check?
This is the XAML declaration of the second form:

```xaml
<Window x:Class="asseXLS.frmVociBase"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"        
        mc:Ignorable="d"
        xmlns:mdix="http://materialdesigninxaml.net/winfx/xaml/themes" 
        xmlns:sl="clr-namespace:asseXLS"        
        Style="{StaticResource MaterialDesignWindow}"
        Background="{DynamicResource MaterialDesignPaper}"
        TextElement.Foreground="{StaticResource MaterialDesignBody}"
        sl:WindowEx.ShowMinimizeButton="false"
        sl:WindowEx.ShowMaximizeButton="false"       
        Title="Configurazione Voci Assenza" Height="600" Width="800" WindowStyle="None" WindowStartupLocation="CenterScreen" ResizeMode="NoResize">
    <Grid>
        <Grid.ColumnDefinitions>
            <ColumnDefinition Width="100"/>
            <ColumnDefinition Width="540"/>
        </Grid.ColumnDefinitions>        
    </Grid>
</Window>
 ```

Thank you, regards.

Roberto

**Comment by Keboo** [comment]
So I am a little surprised that you are seeing the buttons at all. When you set `WindowStyle="None"` it removes the window chrome from around the window. Is it possible that your second window is on top of the first window, and you are seeing the first window with the buttons disabled? This can occur when you use ShowDialog() from the first window.

**Comment by washburnit** [comment]
I didn't consider that "event"...I will check it, thanks for the suggestion.

Roberto
